client/web/src/enterprise/repo/settings/RepoSettingsLogsPage.tsx

```diff
@@ -0,0 +1,190 @@
+import { SettingsAreaRepositoryFields, RepositoryRecordedCommandFields } from '../../../graphql-operations'
+import { LogsPageTabs } from '../../../repo/constants'
+import { eventLogger } from '../../../tracking/eventLogger'
+
+import { useFetchRecordedCommands } from './backend'
+
+import styles from './RepoSettingsLogsPage.module.scss'
+
+export interface RepoSettingsLogsPageProps {
+    repo: SettingsAreaRepositoryFields
+}
+
+/**
+ * The repository settings log page.
+ */
+export const RepoSettingsLogsPage: FC<RepoSettingsLogsPageProps> = ({ repo }) => {
+    const [activeTab, setActiveTab] = useState<number>(LogsPageTabs.COMMANDS)
+    useEffect(() => eventLogger.logPageView('RepoSettingsLogs'))
+    const location = useLocation()
+
+    useEffect(() => {
+        const searchParams = new URLSearchParams(location.search)
+        const tab = searchParams.get('activeTab')
+        if (tab) {
+            const activeTabIdx = parseInt(tab, 10)
+            switch (activeTabIdx) {
+                case LogsPageTabs.SYNCLOGS:
+                    setActiveTab(LogsPageTabs.SYNCLOGS)
+                    break
+                case LogsPageTabs.COMMANDS:
+                default:
+                    setActiveTab(LogsPageTabs.COMMANDS)
+            }
+        }
+    }, [location.search])
+
+    const setActiveTabIndex = (index: number): void => setActiveTab(index)
+
+    return (
+        <>
+            <PageTitle title="Logs" />
+            <PageHeader path={[{ text: 'Logs and activities' }]} headingElement="h2" className="mb-3" />
+
+            <Container>
+                <Tabs
+                    size="medium"
+                    lazy={true}
+                    className={styles.tabContainer}
+                    index={activeTab}
+                    onChange={setActiveTabIndex}
+                >
+                    <TabList>
+                        <Tab>Last repo commands</Tab>
+                        <Tab>Last sync output</Tab>
+                    </TabList>
+
+                    <TabPanels>
+                        <TabPanel>
+                            <LastRepoCommands repo={repo} />
+                        </TabPanel>
+
+                        <TabPanel>
+                            <LastSyncOutput mirrorInfo={repo.mirrorInfo} />
+                        </TabPanel>
+                    </TabPanels>
+                </Tabs>
+            </Container>
+        </>
+    )
+}
+
+interface LastRepoCommandsProps {
+    repo: SettingsAreaRepositoryFields
+}
+
+const LastRepoCommands: FC<LastRepoCommandsProps> = ({ repo }) => {
+    const { recordedCommands, loading, error, fetchMore, hasNextPage } = useFetchRecordedCommands(repo.id)
+
+    return (
+        <>
+            {error && <ErrorAlert error={error} />}
+            <div aria-label="recorded commands">
+                {(!loading && recordedCommands.length) === 0 && <Text className="my-2">No recorded commands yet.</Text>}
```

Comment:
```suggestion
                {(!loading && recordedCommands.length === 0) && <Text className="my-2">No recorded commands yet.</Text>}
```
I guess, closing brace got incorrectly placed?